include/lemlib/movements/turn.hpp

```diff
@@ -50,15 +50,15 @@ class Turn : public Movement {
          */
         float getDist() override;
     private:
-        FAPID angularPID;
+        FAPID<Angle> angularPID;
         std::optional<Pose> targetPose = std::nullopt;
-        Pose startPose = Pose(0, 0, 0);
-        float targetHeading = 0;
+        Pose startPose = Pose(0_in, 0_in, 0_deg);
+        Angle targetHeading = 0_deg;
```

Comment:
```suggestion
        Angle targetHeading = 0_rad;
```
Change to radians for consistency